docs/book/src/user/quick-start.md

```diff
@@ -16,6 +16,7 @@ If using a [provider] that does not support v1beta1 or v1alpha4 yet, please foll
 
 - Install and setup [kubectl] in your local environment
 - Install [kind] and [Docker]
+- Install [Helm]
```

Comment:
nit: I wonder if we should make clear that helm is not always needed

Reply:
I could, I almost did but then thought it'd be easier to maintain if we don't have to update the conditions for installing it when other providers add their own cloud provider or other things that use Helm

Reply:
👍